src/Microsoft.VisualStudio.Editors/Common/ListViewComparer.vb

```diff
@@ -21,9 +21,9 @@ Namespace Microsoft.VisualStudio.Editors.Common
 
         Private _sorting As SortOrder = SortOrder.Ascending
 
-        '@ <Summary>
-        '@  Which column should be used to sort the list. Start from 0
-        '@ </Summary>
+        ''' <Summary>
```

Comment:
❔  Does VB allow `<Summary>` (as opposed to `<summary>`)? This PR has several cases of the former.

Reply:
Yes it does. This is across the entire tree this usage, so I'd do that in one go.

Reply:
https://github.com/dotnet/project-system/pull/3658